server/mediaserver.go

```diff
@@ -62,6 +62,10 @@ var BroadcastJobVideoProfiles = []ffmpeg.VideoProfile{ffmpeg.P240p30fps4x3, ffmp
 var AuthWebhookURL string
 
 var refreshIntervalHttpPush = 1 * time.Minute
+var watchdogResetTimer = func() (context.Context, context.CancelFunc) {
```

Comment:
Technically the term of art for this type of mechanism is ["watchdog"](https://en.wikipedia.org/wiki/Watchdog_timer) which I do find more descriptive with regards to its intent - it periodically checks for activity and closes the session otherwise. (And this particular addition "kicks" the watchdog if we're still mid-activity.) At least, that seems more  descriptive to me than the "refresh interval" that it's associated with - which doesn't actually *refresh* anything.

Anyway, fixed in 8c1e19495b48daeb15c6ab5fa9ba313d686472bc .

Another option is we can keep the "watchdog" terminology (or use something else more descriptive, I'm open to whatever), and rename the "refreshIntervalHttpPush" and related code to associate better with the semantic concept, eg "httpPushWatchdogTimeout".

Reply:
The watchdog terminology + using var names that associate with the semantic concept sound good to me.

Reply:
`httpPushWatchdogTimeout` turned out to be too much of a mouthful (and `watchdogTimeout` is perhaps not specific enough given this code handles a lot of things), so kept your suggestion and renamed `refreshIntervalHttpPush` to `httpPushTimeout`, and added a few comments to clarify its usage. 62be37c4f41e72c6fe0887a725b187c0ae6eda94